common/src/test/java/apoc/ApocConfigTest.java

```diff
@@ -76,7 +77,8 @@ public void setup() throws Exception {
         when(neo4jConfig.expandCommands()).thenReturn(true);
 
         apocConfigFile = new File(getClass().getClassLoader().getResource("apoc.conf").toURI());
-        Files.setPosixFilePermissions(apocConfigFile.toPath(), permittedFilePermissionsForCommandExpansion);
+        apocConfigCommandExpansionFile = new File(getClass().getClassLoader().getResource("apoc-config-command-expansion/apoc.conf").toURI());
```

Comment:
Any reason why not to use the same `apoc.conf` file for this class?

Reply:
Because then I don't need to add and remove the command line from the file for every other test

Reply:
It's more difficult to reason which tests use what config file. I'd rather if some tests depend on a specific file in the filesystem, then we have a file per test class. You could also extract those tests to another class `ApocConfigExpansion`?

Reply:
Okay, sure :)

Reply:
Thanks 😄